middle_end/flambda/types/env/typing_env.rec.ml

```diff
@@ -1322,7 +1332,6 @@ let get_canonical_simple_exn t ?min_name_mode ?name_mode_of_existing_simple
     Simple.pattern_match simple
       ~const:(fun _ -> aliases_with_min_binding_time t)
       ~name:(fun name ~coercion:_ ->
```

Comment:
The coercion argument is marked as ignored but the CR about the coercion being dropped has been removed. Does that mean that the coercion is still dropped or that it will be dealt with later on?

Reply:
In this case, the coercion is still dropped, but that's fine since we're just figuring out which alias tracker and binding time to use.